modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/UpdateEpisodeDetailsTask.kt

```diff
@@ -91,7 +89,6 @@ class UpdateEpisodeDetailsTask @AssistedInject constructor(
                 val downloadUrl = episode.downloadUrl?.toHttpUrlOrNull() ?: continue
                 val request = Request.Builder()
                     .url(downloadUrl)
-                    .addHeader("User-Agent", "Pocket Casts")
```

Comment:
It would be good to check this user agent stays the same. I could be wrong but I have a vague memory that we made the user agent stay the same across all the apps so it was easier for podcasters to track the Pocket Casts platform.

Reply:
This particular user agent was different from the one that we use in all other places. Now it is aligned to `Pocket Casts/Android/<VERSION>`.

Reply:
I changed it in https://github.com/Automattic/pocket-casts-android/pull/3218/commits/69ee862e26105ba8ca3950f29732852629ec9ec6. Now communication with our servers uses the full user-agent and with external ones* `Pocket Casts` one.

*this is mostly true because we still fetch things like chapters through `PodcastCacheService`. But downloads, streaming, and episode details use the simple version.

Reply:
Thanks for doing that. I can see iOS uses "Pocket Casts" [here](https://github.com/Automattic/pocket-casts-ios/blob/trunk/Modules/Server/Sources/PocketCastsServer/Public/Sharing/Structs/ServerConstants.swift#L96).